doc/man1/exekall.1

```diff
@@ -0,0 +1,460 @@
+.\" Man page generated from reStructuredText.
```

Comment:
Mmm don't we want to *not* commit this? Otherwise we'll have to manually keep it in sync with the .rst version in the git repo.

Reply:
If we don't commit that, `man exekall` will not work out of the box. I'd rather not have all users need to build the doc just to get access to man pages, especially since we will turn `lisa-help` in a man page as well pretty soon using that infrastructure.

In order to avoid too much issues, `pedantic_build.sh` will always generate the man page from scratch, so it will follow changes of `exekall --help` and friends without troubles (it's fast to build anyway).